docs/ops/generation/Multinomial_13.md

```diff
@@ -0,0 +1,151 @@
+
+**Versioned name**: *Multinomial-13*
+
+**Category**: *Generation*
+
+**Short description**: *Multinomial* operation generates a sequence of class indices sampled from Multinomial distribution.
+
+**Detailed description**: *Multinomial* operation generates a sequence of class indices sampled from a Multinomial distribution. In this context, the *input* values represent the probabilities associated with each class within the Multinomial distribution. When performing the operation, it randomly selects a class based on these probabilities. Subsequently, the index of the chosen class in the *input* array is appended to the *output* sequence in the corresponding batch.
+
+**Algorithm formulation**:
+
+Given a list of probabilities x1,x2, ..., xn:
+* If *log_probs* is true:
+  * For each probability x, replace it with a value e^x
+* Create a cumulative sum (prefix sum) of those probabilities, ie. create an array of values where the ith value is the sum of the probabilities x1, ..., xi
+* Divide the created array by the final (maximum) value to normalize the cumulative probabilities between [0, 1]
+* Randomly generate a sequence of floats in range [0,1]
+* For each generated number, assign the class with the lowest index for which the cumulative probability is less or equal the generated value.
+* If *replacement* is False (sampling without replacement)
+  * Set the class probability to 0 and adjust the cumulative sum accordingly
+* Convert the output indices to *output_type*
+* Return output indices
+
+**Attributes**:
+
+* ``output_type``
+
+  * **Description**: the type of the output. Determines generation algorithm and affects resulting values. Output numbers generated for different values of *output_type* may not be equal.
+  * **Range of values**: "i32", "i64", "f16", "bf16", "f32", "f64".
+  * **Type**: string
+  * **Required**: *Yes*
+
+* ``replacement``
+
+  * **Description**: controls whether to sample with replacement (classes can be sampled multiple times).
+  * **Range of values**: `true`, `false`
+      * ``true`` - class indices can be sampled multiple times.
+      * ``false`` - class indices will not repeat in the output and size of *input*'s ``class_size`` dimension is required to be larger or equal to *num_samples* value.
+  * **Type**: `bool`
+  * **Required**: *Yes*
+
+* ``log_probs``
+
+  * **Description**: allows to control whether *inputs* should be treated as probabilities or unnormalized log probabilities.
+  * **Range of values**: `true`, `false`
+      * ``true`` - set values in *inputs* are unnormalized log probabilities that can be any real number.
+      * ``false`` - probabilities in *inputs* are expected to be non-negative, finite and have non-zero-sum.
+  * **Type**: `bool`
+  * **Required**: *Yes*
+
+* ``global_seed``
+
+  * **Description**: global seed value.
+  * **Range of values**: positive integers
+  * **Type**: `int`
```

Comment:
```suggestion
  * **Type**: `unsigned int`
```
As value range is [0 max]. 
the int or unsigned int has platform dependent bit-length, maybe this value should have fixed bit-length?

Same comments for `op_seed`.